Makefile

```
@@ -1,6 +1,6 @@
 SRC=*.go cmd/terminal-to-html/*.go
 BINARY=terminal-to-html
-BUILDCMD=go build -o $@ cmd/terminal-to-html/*
+BUILDCMD=go build -o $@ cmd/terminal-to-html/*.go
```

Hmm. That looks odd. Would this work?

```suggestion
BUILDCMD=go build -o $@ ./cmd/terminal-to-html
```

---

```
main module (github.com/buildkite/terminal-to-html/v3) does not contain package github.com/buildkite/terminal-to-html/v3/cmd/terminal-to-html
make: *** [terminal-to-html] Error 1
```

Nope

---

This was due to the extra module, which is apparently not supposed to be there, have removed it.